source/includes/Concepts/concept.md

```diff
@@ -93,16 +89,15 @@ GET /concept?
     
 * ### Query concept by UUID.
 
-    Retrieve an concepts by its UUID. Returns a `404 Not Found` status if concepts not exists. If user not logged 
-    in to perform this action, a `401 Unauthorized` status returned.
+    Retrieve a concept by its UUID. Returns a `404 Not Found` status if concepts not exist. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
     
 ```console
 GET /concept/:target_concept_uuid
 ```
    
 ## Create a concept
 
-* To Create an concept you need to specify below attributes in the request body.If you are not logged in to perform this action,
+* To Create a concept, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

Comment:
```suggestion
* To create a concept, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

Reply:
@sanjulamadurapperuma all the operations were starting from a capital letter so should I change them all like update delete as well ?

Reply:
I think it is better to because it actually is a verb, not the operation name. @dkayiwa @Akayeshmantha appreciate your thoughts on this.